apps/api/src/app/feeds/usecases/create-feed/create-feed.usecase.ts

```diff
@@ -9,6 +9,15 @@ export class CreateFeed {
   constructor(private feedRepository: FeedRepository, private createChange: CreateChange) {}
 
   async execute(command: CreateFeedCommand): Promise<FeedEntity> {
+    const feedExist = await this.feedRepository.findOne({
+      _organizationId: command.organizationId,
```

Comment:
Feeds are per enviroment, sou would need to add here _environmentId instead of organizationId